cmake/GtsamMakeConfigFile.cmake

```diff
@@ -20,13 +20,32 @@ function(GtsamMakeConfigFile PACKAGE_NAME)
 		set(EXTRA_FILE "_does_not_exist_")
 	endif()
 
+	# Version file
+	if(NOT DEFINED ${PACKAGE_NAME}_VERSION AND DEFINED ${PACKAGE_NAME}_VERSION_STRING)
+		set(${PACKAGE_NAME}_VERSION ${${PACKAGE_NAME}_VERSION_STRING})
+	endif()
+
```

Comment:
What were these lines for? I mean, in what case will `${PACKAGE_NAME}_VERSION` be defined in this script, which is used while compiling gtsam "locally" (i.e. not for users)?

Reply:
I noticed that this file is installed on `lib/cmake/GTSAMCMakeTools/`. From what I understood `GTSAMCMakeTools` are tools that were conceived for GTSAM, but that can be used for other projects as well.

These lines exist because GTSAM sets it's version string on `GTSAM_VERSION_STRING`, while in CMake the version variable is `<PackageName>_VERSION`, and the tools are supposed to work internally and externally if used by other projects.